types/wlr_output_layout.c

```diff
@@ -328,7 +328,7 @@ void wlr_output_layout_closest_point(struct wlr_output_layout *layout,
 		return;
 	}
 
-	double min_x = DBL_MAX, min_y = DBL_MAX, min_distance = DBL_MAX;
+	double min_x = 0, min_y = 0, min_distance = DBL_MAX;
```

Comment:
What happens if the minimum x coord is > 0? This will incorrectly set it as zero, right?

Reply:
No, because you compare `min_distance` first, and `min_distance` is DBL_MAX initially. So any point will be closer to a rectangle than this.

Reply:
Oh right, the min x/y are not used in the loop, only the min distance is used.